### PR TITLE
[DOC] add badges and link to doc

### PR DESCRIPTION
**Pull request by Remi-Gau**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
 project = 'BIDS Validator'
-copyright = '2024, BIDS Contributors'
```
Comment (Remi-Gau) [inline docs/conf.py:10]: should not be needed 😉

File: README.md [hunk 2]
```diff
@@ -76,3 +80,8 @@ To modify the schema a clone of bids-standard/bids-specification will need to be
 After changes to the schema have been made to a local copy the dereferenced single json file used by the validator will need to be built. The `bidsschematools` python package does this. It can be installed from pypi via pip or a local installation can be made. It lives in the specification repository here https://github.com/bids-standard/bids-specification/tree/master/tools/schemacode
 
 The command to compile a dereferenced schema is `bst -v export --output src/schema.json` (this assumes you are in the root of the bids-specification repo). Once compiled it can be passed to the validator via the `-s` flag, `./bids-validator-deno -s <path to schema> <path to dataset>`
+
+
+## Documentation
+
+The BIDS validator documentaion is available on [Read the Docs](https://bids-validator.readthedocs.io/en/latest/).
```
Comment (Remi-Gau) [inline README.md:87]:
```suggestion
The BIDS validator documentation is available on [Read the Docs](https://bids-validator.readthedocs.io/en/latest/).
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+[![Deno build](https://github.com/bids-standard/bids-validator/actions/workflows/deno_tests.yml/badge.svg)](https://github.com/bids-standard/bids-validator/actions/workflows/deno_tests.yml)
+[![Web validator](https://github.com/bids-standard/bids-validator/actions/workflows/web_build.yml/badge.svg)](https://github.com/bids-standard/bids-validator/actions/workflows/web_build.yml)
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3688707.svg)](https://doi.org/10.5281/zenodo.3688707)
```
Comment (effigies) [inline README.md:3]: Can we add a badge for the docs?

Comment (Remi-Gau) [reply README.md:4]: done

**Pull request merged**